src/util/CountryUtil.js

```diff
@@ -39,6 +39,27 @@ fn.getCountries = function() {
   return sorted;
 };
 
+fn.getCountryCode = function(){
+
+  const countries = _.omit(bundles.country, 'HM', 'BV', 'TF');
+  // HM, BV, and TF do not have phone prefixes, so don't give the
+  // user the option to choose these countries. FYI it appears that these
+  // countries do not have calling codes because they are ~~uninhabited~~
+
+  let collection = _.map(countries, function(name, code) {
+    return { name: name, code: code };
+  });
+
+  collection = _.sortBy(collection, 'name');
+  const sorted = {};
+
+  _.each(collection, function(country) {
+    sorted[country.code] = country.name +' (' + country.code + ')';
```

Comment:
This formatting isn't applicable to all of our supported languages. Please add this to our `login.properties` file with a comment describing the intended structure:

```properties
# {0} is the name of the country and {1} is the country code
country.option.label = {0} ({1})
```

Reply:
added and updated.